Avatar/avatar.css

```diff
@@ -0,0 +1,141 @@
+.header{
+    display: flex;
+    justify-content: space-between;
+    background-color:rgb(139, 185, 179);
+    color:black;
+    font-weight: bold;
+    font-size:19px;
+}
+.home{
+    display: flex;
+    gap:0.5rem;
+}
+#home-link{
+    margin-top:1rem;
+    text-decoration: none;
+    color:black;
+}
+.container{
+    width:100%;
+    display: flex;
+    gap: 7rem;
+}
+.sidebar-container{
+    display: flex;
+    flex-direction: column;
+    gap: 3rem;
+    width: 150px;
+    margin-top: 1rem;
+    padding: 0rem 1rem 1rem 1rem;
+    border-radius: 1rem;
+    border: 1px solid black;
+    background-color:rgb(236, 189, 189);
+}
+#side-heading{
+    font-size:22px;
+    font-weight: bold;
+    margin-bottom:-0.5rem;
+}
+#special-btn{
+    color: black;
+    text-decoration: underline;
+}
+.componant{
+    text-decoration: none;
+    color:grey;
+    font-weight: bold;
+    font-size: 20px;
+}
+.componant:hover{
+    text-decoration: underline;
+    color: black;
+}
+#ambani1{
+    width:100px;
+    height: 100px;
+    border-radius:50%
+}
+#ambani2{
+    width:75px;
+    height: 75px;
+    border-radius:50%
+
+}
+#ambani3{
+    width: 50px;
+    height: 50px;
+    border-radius:50%
+}
+#ambani4{
+    width: 40px;
+    height: 40px;
+    border-radius:50%
+}
```

Comment:
I would suggest not use ID , keep a generic class for different sizes. `avatar-sm' , `avatar-md`.

Avoid using pixels , use rems

`border-radius:50%` is repeating , you could keep a generic class for that

Reply:
okk will do that.